data/defaults.yaml

```diff
@@ -127,7 +127,7 @@ prometheus::process_exporter::group: 'process-exporter'
 prometheus::process_exporter::package_ensure: 'latest'
 prometheus::process_exporter::package_name: 'process-exporter'
 prometheus::process_exporter::user: 'process-exporter'
-prometheus::process_exporter::version: '0.1.0'
+prometheus::process_exporter::version: '0.2.12'
```

Comment:
for now we would like to stay on version ```0.1.0```. Bumping it would be a new major release. We want to do a new major release later where we bump all versions.